test/integration/harness/lib/docker_ctl.py

```
@@ -0,0 +1,54 @@
+from pathlib import Path
+
+COMPOSE_FILE = (
+    Path(__file__).resolve().parents[2]
+    / "docker"
+    / "docker-compose.yml"
+)
+
+
+def _compose(*args: str) -> subprocess.CompletedProcess:
+    cmd = ["docker", "compose", "-f", str(COMPOSE_FILE), *args]
+    return subprocess.run(cmd, check=True, capture_output=True, text=True)
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)